Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -104,7 +104,7 @@ protected function configure()
                     new InputOption('config-file', '', InputOption::VALUE_OPTIONAL, 'The path to a .php_cs file ', null),
                     new InputOption('dry-run', '', InputOption::VALUE_NONE, 'Only shows which files would have been modified'),
                     new InputOption('level', '', InputOption::VALUE_REQUIRED, 'The level of fixes (can be psr0, psr1, psr2, or symfony (formerly all))', null),
-                    new InputOption('using-cache', '', InputOption::VALUE_REQUIRED, 'Does cache should be used (can be yes or no)', null),
+                    new InputOption('using-cache', '', InputOption::VALUE_OPTIONAL, 'Set cache should be used (can be yes or no)', 'yes'),
```

Comment:
Thank you for pointing that out. I did miss that use case and changed the default back to `null`.